### PR TITLE
Server routes

### PR DESCRIPTION
**Pull request by yukiej**
Hey @mrkent , could you review the API in server/index.js please? I also made some small changes to the database schema and reviews seeding script to make sure each review had all the different kinds of star ratings, not just a single star rating. Thank you!

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
 
-app.listen(port, () => console.log(`Server listening on port ${port}`));
\ No newline at end of file
+
+app.get('/rooms/:homeid/reviews', (req, res) => {
+  const homeID = req.params.homeid;
+  const keyword = req.query.keyword || undefined;
+  // get data from database
+  if (keyword === undefined) {
+    helpers.getReviews(homeID, (results) => {
```
Comment (mrkent) [inline server/index.js]: Seems like if you get an error from `queryHelpers.js`, this request would just hang, and no response ever provided to client. I'm currently doing the same thing, so not sure what's the best way to handle it. would like to discuss.

File: server/index.js [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
 const bodyParser = require('body-parser');
+const path = require('path');
 const db = require('../database/index.js');
+const helpers = require('./queryHelpers.js');
 
 const app = express();
 const port = 3002;
 
+// app.use(express.static('../public'));
+app.use(express.static(`${__dirname}/../client/dist/`));
```
[the remark this answers is not in the document]
Comment (yukiej) [reply server/index.js]: Got it, thanks!

File: server/index.js [hunk 3]
```diff
@@ -1,12 +1,40 @@
...
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
-app.use(express.static('../public'));
 
-app.listen(port, () => console.log(`Server listening on port ${port}`));
\ No newline at end of file
+
+app.get('/rooms/:homeid/reviews', (req, res) => {
+  const homeID = req.params.homeid;
+  const keyword = req.query.keyword || undefined;
```
[the remark this answers is not in the document]
Comment (yukiej) [reply server/index.js]: Thanks!

File: server/index.js (hunk 1 above)
Comment (yukiej) [reply server/index.js]: Thanks for the helpful discussion! I've added error handling in index.js and made sure errors are passed to the callbacks in the helper functions.

**Comment by yukiej** [comment]
@mrkent I've made the changes you suggested! Could you look at the most recent commit to make sure it looks okay to you before the merge? Thanks!

**Comment by yukiej** [comment]
@mrkent Updated error handling to pass errors to next instead of throwing them! Thanks for catching that (no pun intended). Let me know if it looks ok to merge!

**Pull request merged**